enterprise/__init__.py

```diff
@@ -2,6 +2,6 @@
 Your project description goes here.
 """
 
-__version__ = "3.44.0"
+__version__ = "3.43.1"
```

Comment:
This'll fail when creating new PyPI release since the tag will already be present on PyPI and the latest version present on PyPI will still remain to be `3.44.0`. 
Since platform has a manual constraint already present so this shouldn't be needed for handling platform upgrade.